torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -2011,6 +2030,36 @@ RegisterOperators reg2({
           push(stack, std::sqrt(a));
           return 0;
         }),
+    DEFINE_GENERI_PYTHON_OP(
```

Comment:
I think, given your macro, defining an op should look like this:
```suggestion
    DEFINE_GENERI_PYTHON_OP(aten::isnan, std::isnan(a), std::isnan(a), float)
```

Reply:
I based this on they did it on line 1847 but im open, thoughts @eellison